plugins/terraform/data_source_layer0_loadbalancer.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/quintilesims/layer0/cli/client"
+)
+
+func dataSourcelayer0LoadBalancer() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourcelayer0LoadBalancerRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"environment_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"environment_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"private": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+			"url": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"service_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"service_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"healthcheck_healthy_threshold": {
```

Comment:
Having healthcheck data seems overkill in my opinion - I'd vote for removing it. 
If we keep it, then we should follow the existing pattern: https://github.com/quintilesims/layer0/blob/develop/plugins/terraform/resource_load_balancer.go#L113